### DISCUSSION TITLE
Does o3de support the development of 2D games?

### DISCUSSION THREAD

**Discussion by C-BJ** [o3de/o3de#11171] [Q&A] [answered]
Does o3de support the development of 2D games?

**Comment by VickyAtAZ** [comment, marked as answer]
o3de has an 2d UI editor which is intended to be used for make 3d game's 2d menu and in-game UI. It's possible you can develop some 2d games with it. The main o3de editor doesn't have components which are designated for 2d games. 
Overall, I think it's better to use some 2d game engine to develop 2d games because it would be more dedicated and light weight. 
If you want to develop a game with 3d models but 2d game play and orthographic camera, o3de would be a good choice.
